test/lightning/policies/project_user_permissions_test.exs

```diff
@@ -56,6 +69,14 @@ defmodule Lightning.Policies.ProjectUserPermissionsTest do
       assert ProjectUsers |> Permissions.can?(:access_project, viewer, project)
     end
 
+    test "can not access that scheduled deletion project", %{
```

Comment:
"can not access a project that is scheduled for deletion"

Reply:
Done